### PR TITLE
make zed.rc values reflect their default value

### PR DESCRIPTION
**Pull request by kpande**
this helps avoid confusion if a user expects functionality to be enabled.

Signed-off-by: Kash Pande <kash@tripleback.net>

### Motivation and Context
I thought that scrub after resilver was enabled by default, since it is commented out with a value of 1. The default is 0. This was undocumented.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Performance enhancement (non-breaking change which improves efficiency)
- [ ] Code cleanup (non-breaking change which makes code smaller or more readable)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (a change to man pages or other documentation)

### Checklist:

- [x] My code follows the ZFS on Linux [code style requirements](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#coding-conventions).
- [x] I have updated the documentation accordingly.
- [x] I have read the [**contributing** document](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md).
- [ ] I have added [tests](https://github.com/zfsonlinux/zfs/tree/master/tests) to cover my changes.
- [x] All new and existing tests passed.
- [x] All commit messages are properly formatted and contain [`Signed-off-by`](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md#signed-off-by).

### CODE REVIEW

File: cmd/zed/zed.d/zed.rc [hunk 1]
```diff
@@ -54,7 +54,7 @@
 # Send notifications for 'ereport.fs.zfs.data' events.
 # Disabled by default
 #
-#ZED_NOTIFY_DATA=1
+#ZED_NOTIFY_DATA=0
```
Comment (rlaager) [inline cmd/zed/zed.d/zed.rc]: It looks like the "default" here is more the empty string than `0`. `cmd/zed/zed.d/data-notify.sh` has: `[ -n "${ZED_NOTIFY_DATA}" ] || exit 3`. So it looks like `0` would have the same effect as `1`.

Comment (kpande) [reply cmd/zed/zed.d/zed.rc]: interesting. good catch. I saw elsewhere it checked for 1.

Comment (rlaager) [reply cmd/zed/zed.d/zed.rc]: I think you have this backwards. ZED_NOTIFY_DATA is the one that should be the empty string. Arguably ZED_SCRUB_AFTER_RESILVER could be the empty string too, but 0 is okay for that one.

**Comment by kpande** [comment]
jeez, wet sock brain today. I pushed the fixes for that, so that notify data is empty and scrub after resilver is 0.

**Pull request merged**